cfn.yaml

```diff
@@ -1,112 +1,127 @@
 Mappings:
```

Comment:
Please exclude cfn.yaml file. Not sure when it got into the repo but it should be in the .gitignore.